### ISSUE TITLE
Doesn't work correctly with SSR (Nuxt)

### ISSUE THREAD

**Issue by kodekrendel** [ccforward/progressive-image#3]
Hi there.
Beautiful package, but I have an issue that I can't figure out. Appreciate your help.

I am using Nuxt, and progressive-image workes only on initial page load. If I use Back browser button it load only blured image.

To see what I mean:

1. Go to http://messy-straw.surge.sh/images/ Wait till images load.
2. Then go to Main Page.
3. And then return to previous page with Back browser button. Images doesn't lazyload again.
4. In all other use cases code is working fine, bug appears only when using Back

I import progressive-image as a Nuxt plugin:

```
import Vue from 'vue'
import progressive from 'progressive-image/dist/vue'

Vue.use(progressive, {
  removePreview: true
})
```
Then I just use it in `.vue` file like in your example.

**Comment by krisnaw** [comment]
I have got a different issue, the image preview won't disappear even if I set the removePreview value to 'true'.

**Comment by hxzAugustinus** [comment]
add  this
`
  <keep-alive>
      <router-view></router-view>
    </keep-alive>
`
keep the image view state  will  work  fine.

**Comment by kodekrendel** [comment]
@hxzAugustinus where to add this, can you be more specific?

**Comment by hxzAugustinus** [comment]
@kodekrendel     the Website [  http://messy-straw.surge.sh/images/]( http://messy-straw.surge.sh/images/)  doesn.t  work,so in general you search  **router-view**  in app.vue ,then 
include it  with  keep-alive

**Comment by kodekrendel** [comment]
> so in general you search router-view in app.vue ,then include it with keep-alive.

I don't understood a single word. What is router-view, where I must look for it, how to include keep-alive, what is keep-alive.

**Comment by kodekrendel** [comment]
Here - now it doesn't work entirely - http://furry-spring.surge.sh/

**Comment by hxzAugustinus** [comment]
because you forget to  import  "./node_modules/progressive-image/dist/index.css"

**Comment by kodekrendel** [comment]
Yeah, but problem remains. Images doesn't load when use Back button.
